src/engraving/dom/chord.cpp

```diff
@@ -3231,6 +3231,11 @@ Ornament* Chord::findOrnament() const
             return toTrill(spanner)->ornament();
         }
     }
+    for (Spanner* spanner : m_endingSpanners) {
+        if (spanner->isTrill()) {
+            return toTrill(spanner)->ornament();
+        }
+    }
```

Comment:
this even seems to work for trill spanners that start on previous note and extend to next note, surprisingly.

Reply:
Please attach your score then, this definitely fixed it for me

Reply:
I couldn't find/know how to add a file with *.mscz extension here. I am sending you as a screenshot. But in the meantime I discovered something. When I save the file and open it again, it does not make any errors, but when I change the trill/properties/ornament/interval settings, it does not play correctly.
![screens](https://github.com/musescore/MuseScore/assets/35005568/9f4c2f40-2fbf-4fd7-b379-e3d0d84814d2)

Reply:
Just rename the .mscz to .zip (or anything really).

Reply:
Anyway I've pushed a separate fix that ensures after editing a trill the playback for the whole length of the trill is updated (vs just that for the note its attached to).

Reply:
Turns out it's important NOT to return the ornament if it's being queried for layout purposes (particularly with regard to auxiliary notes to indicate trill interval).